jest.conf/testUtils.js

```diff
@@ -1,5 +1,9 @@
 import { setMedia } from 'mock-match-media';
 
+const percySnapshot = require('@percy/puppeteer');
```

Comment:
Can't we use the import from syntax here?

Reply:
Sorry I had overlooked this. Will update it to use the import from syntax here :+1: